### ISSUE TITLE
Did v5 tag got re-tagged?

### ISSUE THREAD

**Issue by polesen** [php-actions/composer#47]
We've been running against the `v5` tag for at least a month, and suddenly all our builds started failing with errors on accessing private repos. Looking at the repo, it looks like the `v5` tag is only a couple of days old now. Did the old tag got deleted and re-tagged on another sha?

**Comment by Apfelfrisch** [comment]
Second this. Since yesterday all our builds are failing with `%git@github.com: Permission denied (publickey).
215
fatal: Could not read from remote repository.Please make sure you have the correct access rights and the repository exists.`

**Comment by polesen** [comment]
I've falled back to running against a specific git sha now, because `v4` doesn't support the extension installing we were using:

```
uses: php-actions/composer@ff923f6d2bca1f888115c734de8cc64147bb2abc
```

**Comment by g105b** [comment]
Hi @polesen ,

There was indeed a release to v5, but I have reverted that release now and said the changes will be made to v6 to avoid any downtime for anyone. To confirm: v5 points at `ff923f6d2bca1f888115c734de8cc64147bb2abc` again.

The reason for this re-tagging is how Github Actions releases are handled slightly differently. I made the call to make a patch release to v5 to fix a small bug in regards to environment variables, but as this has had an unexpected side effect I have reverted back and will try to come up with a much more concrete release strategy.

Thanks for pointing this out to me. I'll close the issue for now since it's been reverted.

Greg.
